src/visualization/plot_recipes.jl

```diff
@@ -0,0 +1,292 @@
+# Convenience type to allow dispatch on ODESolution objects that were created by Trixi
+#
+# Note: This is an experimental feature and may be changed in future releases without notice.
+const TrixiODESolution = ODESolution{T, N, uType, uType2, DType, tType, rateType, P} where
+    {T, N, uType, uType2, DType, tType, rateType, P<:ODEProblem{uType_, tType_, isinplace, P_} where
+     {uType_, tType_, isinplace, P_<:AbstractSemidiscretization}}
```

Comment:
Maybe we should include `Trixi.rhs!` here for the RHS type to be more specific?

Reply:
I do not understand how this would improve things, but I accept that you know know this stuff much better than I do :-) However, I have no clue how to do this - could you maybe provide a suggestion?

Reply:
Hmm, the idea is that we're already committing type piracy by defining methods from some packages for types of other packages using this part. Although that's okay here, I think, since we already include our own types here, it might be better to be even more specific.

Reply:
Changed in b3bba138

Reply:
Great, thanks!